### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by ColleagueRiley**
Added credit + fixed workspace badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ To compile and run all of the examples in a row, you can use `make runExamples`
 - [hidefromkgb's 'mac_load' repository](https://github.com/hidefromkgb/mac_load#objective-c-gui-app-example) - the backend for some of the important parts of Silicon (such as defining Objective-C types and functions).
 - Apple - all of the headers from [include/Silicon/headers](include/Silicon/headers/) have been directly copied and modified to not have any Objective-C shenanigans in them for them to be compiled in C.
 - EimaMei - Original creator of Silicon
-- [Colleague Riley](https://github.com/ColleagueRiley) - Creator of the orginal single-header version of Silicon (silicon.h) and co-author of silicon
+- [Colleague Riley](https://github.com/ColleagueRiley) - Creator of the original single-header version of Silicon (silicon.h) and co-author of silicon
```
Comment (EimaMei) [inline README.md]: Fix the capitalization, that being changing 'Creator' to be in lowercase and 'silicon' to have its first letter capitalized. Also a dot is required at the end of the sentence.

**Review by EimaKve — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -82,8 +82,8 @@ To compile and run all of the examples in a row, you can use `make runExamples`
 ## General
 - [hidefromkgb's 'mac_load' repository](https://github.com/hidefromkgb/mac_load#objective-c-gui-app-example) - the backend for some of the important parts of Silicon (such as defining Objective-C types and functions).
 - Apple - all of the headers from [include/Silicon/headers](include/Silicon/headers/) have been directly copied and modified to not have any Objective-C shenanigans in them for them to be compiled in C.
-- EimaMei - Original creator of Silicon
-- [Colleague Riley](https://github.com/ColleagueRiley) - Creator of the original single-header version of Silicon (silicon.h) and co-author of silicon
+- EimaMei - original creator of Silicon
```
Comment (EimaMei) [inline README.md]: You forgot a dot here.

File: README.md [hunk 3]
```diff
@@ -82,8 +82,8 @@ To compile and run all of the examples in a row, you can use `make runExamples`
...
+- [Colleague Riley](https://github.com/ColleagueRiley) - creator of the original single-header version of Silicon (Silicon.h) and co-author of Silicon
```
Comment (EimaMei) [inline README.md]: You forgot a dot here.

**Review by EimaKve — CHANGES_REQUESTED** [review]
Dots are required at the end of the sentences

**Review by EimaKve — APPROVED** [review]

**Pull request merged**